torchao/quantization/quantize_/common/packing_format.py

```diff
@@ -40,3 +40,6 @@ class PackingFormat(str, Enum):
     Unpacked means the subbyte quantized data is stored as int8
     """
     UNPACKED_TO_INT8 = "unpacked_to_int8"
+
+    "int4_xpu_int_zp is referring to the format used by int4 weight-only quantization on XPU with int zero point, which is a groupwise quantization format."
+    INT4_XPU_INT_ZP = "int4_xpu_int_zp"
```

Comment:
please don't include int4 and xpu in the name, can you name this in terms of of how the quantized data is packed?

Reply:
The int4 weight xpu is a plain format tensor according  to this doc, it just pack 2 int4 weight elements in a byte. So I change it to the plain.

Reply:
@liangan1 can you use `PLAIN_INT32` for packing_format, and rename things accordingly (tensor subclass, files etc.)

Reply:
Done. change it to Int4PlainInt32. pls help to review again.